src/Jackett.Common/Indexers/HDBitsApi.cs

```diff
@@ -103,11 +104,16 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                     ["id"] = query.TvdbID
                 };
 
-                if (query.Season != 0)
-                    requestData["tvdb"]["season"] = query.Season;
+                if (DateTime.TryParseExact($"{query.Season} {query.Episode}", "yyyy MM/dd", CultureInfo.InvariantCulture, DateTimeStyles.None, out var showDate))
+                    requestData["search"] = showDate.ToString("yyyy-MM-dd");
+                else
```

Comment:
most sites that have daily series use yyyy.MM.dd on their episode titles. How did you arrive at yyyy-MM-dd ?

Reply:
Looks like that's what Sonarr have been using for it since 2015. Maybe HDBits use their own formatting, because reasons?

Reply:
just wondering, it there is such a variety of episode date formats, if we shouldn't amend the torznab query.GetEpisodeSearchString() to have an optional format parm, so we can handle dots, dashes and spaces in the returned string.
https://github.com/Jackett/Jackett/blob/38a3314f6fb83afe63e99f4d0b72a4fba1f527ad/src/Jackett.Common/Models/TorznabQuery.cs#L208-L231
this torznab routine effectively does what is hardcoded in this indexer's PR